### PR TITLE
[css-font-loading] Add a test for FontFaceSet.has()

### PR DESCRIPTION
**Pull request by litherum**
We're failing this test in WebKit, and I'm fixing WebKit in https://bugs.webkit.org/show_bug.cgi?id=229848

### CODE REVIEW

File: css/css-font-loading/fontfaceset-has.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    src: url("resources/Rochester.otf") format("opentype");
+}
+</style>
+</head>
+<body>
+<script>
+test(function(t) {
+    let fonts = document.fonts;
+    let font = fonts.keys().next().value;
+    let font2 = new FontFace("WebFont2", "url('resources/Rochester.otf') format('opentype')");
```
Comment (smfr) [inline css/css-font-loading/fontfaceset-has.html:22]: Is using a font with the same name part of the test, or would it be better to distinguish them?

**Review by smfr — APPROVED** [review]

**Review by jfkthame — APPROVED** [review]

**Review by drott — APPROVED** [review]
LGTM, thanks for adding that.

**Review by tabatkins — APPROVED** [review]

File: css/css-font-loading/fontfaceset-has.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
 <script>
 test(function(t) {
     let fonts = document.fonts;
-    let font = fonts.keys().next().value;
+    let font = [...fonts.keys()][0];
```
Comment (jfkthame) [inline css/css-font-loading/fontfaceset-has.html:21]: @tabatkins Was there a specific reason this change is needed? Unfortunately, it causes the test to fail in Gecko (with "fonts.keys() is not iterable"), although that's not the feature this test is meant to be addressing. Can we revert to Myles' previous syntax here?

The fact that `[...fonts.keys()]` doesn't work in Gecko would appear to be a separate interop issue that deserves its own test; it shouldn't be mixed up with whether `.has()` works.

**Comment by tabatkins** [comment]
I just found the changed syntax vastly more readable; that's absolutely a Firefox bug if it doesn't work, since it's clearly iterable. I don't like relying on `.next()` existing for iterables, since it doesn't for a lot of them.

This wasn't an *important* change, tho, and we can change it back if it's causing problems, tho.

**Pull request merged**